coral/src/app/features/components/SchemaRequestDetails.tsx

```diff
@@ -27,10 +30,32 @@ function SchemaRequestDetails(props: DetailsModalContentProps) {
         <dd>{request.topicname}</dd>
       </Box.Flex>
 
-      <GridItem colSpan={"span-2"}>
+      <Box.Flex flexDirection={"column"}>
```

Comment:
I don't think using flex instead of grid here has any benefit. Also, the current layout looks a little weird with all the white space on the side:
<img width="837" alt="Screenshot 2023-10-19 at 14 24 44" src="https://github.com/Aiven-Open/klaw/assets/20607294/46fb7f62-26f7-4104-9c91-0c85cfd90357">

An `GridItem` (soon to be deprecated an replaced by Grid.Item` may be something like this:

```tsx
      <GridItem colSpan={"span-2"}>
        <Label>Schema version</Label>
        <dd>{request.schemaversion}</dd>
      </GridItem>
      {request.forceRegister && (
        <GridItem colSpan={"span-2"}>
          <Label>Force register</Label>
          <dd>
            {" "}
            <Typography.Small>
              Warning: This schema is being force registered. This will override
              standard validation process of the schema registry.{" "}
              <a
                target="_blank"
                rel="noreferrer"
                href={
                  "https://www.klaw-project.io/docs/HowTo/schemas/Promote-a-schema/#how-does-force-register-work"
                }
              >
                Learn more
              </a>
            </Typography.Small>
          </dd>
        </GridItem>
      )}
```

<img width="841" alt="Screenshot 2023-10-19 at 14 23 09" src="https://github.com/Aiven-Open/klaw/assets/20607294/dc5713e0-5308-466c-ab51-1a8dc53e0439">


Or even like this:

```tsx
           <GridItem>
        <Label>Schema version</Label>
        <dd>{request.schemaversion}</dd>
      </GridItem>
      <GridItem>
        <Label>Force register</Label>
        <dd>
          {" "}
          <Typography.Small>
            Warning: This schema is being force registered. This will override
            standard validation process of the schema registry.{" "}
            <a
              target="_blank"
              rel="noreferrer"
              href={
                "https://www.klaw-project.io/docs/HowTo/schemas/Promote-a-schema/#how-does-force-register-work"
              }
            >
              Learn more
            </a>
          </Typography.Small>
        </dd>
      </GridItem>
```

Which renders this:

<img width="809" alt="Screenshot 2023-10-19 at 14 27 09" src="https://github.com/Aiven-Open/klaw/assets/20607294/6efcfadb-8ae3-4341-b767-ac56a152291f">

Reply:
Hu, that's weird that it's broken up for you in that way, it looked like in my screenshot 🤔 I'll check again and also use Grid.Item. To be honest, I just copy and pasted what we did in the row above without thinking much about it 🙈

Reply:
Replaced Box.Flex with GridItem! Maybe you can check again how it looks for you, because my version looked different than yours before, too 🤔 

https://github.com/Aiven-Open/klaw/pull/1903/commits/f7916dbf368f80c79890199f4c0ca6fa8fe00a09